sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientTests.cs

```diff
@@ -192,5 +193,24 @@ public void MonitorQueryModelFactory_LogsQueryResult_ConvertBinaryDataToJsonElem
             Assert.AreEqual("PartialError", result.Error.Code);
             Assert.AreEqual("There were some errors when processing your query.", result.Error.Message);
         }
+
+        [Test]
+        public void ValidateMonitorModelFactoryTableCreation()
+        {
+            LogsTableColumn logsTableColumn1 = MonitorQueryModelFactory.LogsTableColumn("column1", LogsColumnType.String);
+            LogsTableColumn logsTableColumn2 = MonitorQueryModelFactory.LogsTableColumn("column2", LogsColumnType.String);
+            LogsTableColumn[] logsTableColumns = new LogsTableColumn[] { logsTableColumn1, logsTableColumn2 };
+            Object[] rowValues = new Object[] { "row1", "row2" };
```

Comment:
Let's add different data types to the array and make sure that all of them can be retrieved.

Similar to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs#L263-L295